charts/s1-agent/templates/common/configmaps.yaml

```diff
@@ -30,3 +30,19 @@ data:
         configMap:
           name: {{ include "agent.fullname" . }}
 {{ end }}
+
+---
+
+{{ if or (not .Values.configuration.env.injection.enabled) (eq (include "serverlessOnlyMode" .) "false") }}
```

Comment:
Shouldn't it be "and" instead of "or"?
I guess the first condition and the second one should usually have the same value tho

Reply:
no, we allow it for any k8s flavor but pure eks fargate.

Reply:
Consider putting a comment to make it clear that's what we intend to check